custom_metrics/security.js

```diff
@@ -0,0 +1,21 @@
+return JSON.stringify({
+	"iframe-allow": [].map.call(document.querySelectorAll('iframe[allow]'), (x) => {
+		return {
+			"allow": x.getAttribute('allow'),
+			"src": x.getAttribute(src)
```

Comment:
Do you actually need the full `src` value? Not sure how useful that will be. If not needed you could return an array of `allow` values instead.

Reply:
I wanted a way to uniquely identify `<iframe>` elements such that - if need-be - we could determine which `<iframe>`s had both `allow` and `sandbox` attribute. I think we should at least get the host of `src`.

I could change code so that we get both `allow` and `sandbox` attributes in one metric, and just get the host of the src, would that be better?

Reply:
In that case maybe consider combining them into something like an array of attributes per element.

```js
"iframe-allow-sandbox": Array.from(document.querySelectorAll('iframe[sandbox], iframe[allow]')).map(iframe => {
  return {
    "sandbox": iframe.getAttribute('sandbox'),
    "allow": iframe.getAttribute('allow')
  }
}),
```

Reply:
I added both `sandbox` and `allow` in one metric. I also added the origin, as I suspect that many iframes that have a `sandbox` attribute were added by third-party code, e.g. Google Ads (which paints a different picture than if web developers added this attribute themselves)